modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -13,20 +13,16 @@
 	--n-tabs-display: flex;
 	--n-tabs-direction: column;
 	--n-tabs-gap: 10px;
-
 	--n-tabs-heading-display: flex;
```

Comment:
Remove empty lines.

Reply:
This isn't really up to me, but I would say that these vars are grouped into distinct groups - it may be for readability of the dev working on this that they are separated.

The sass will get compiled and the line breaks auto removed, so then it's just a coding standard, and developer preference question.

What do you think?

Reply:
"developer preference" 😂
It just occurred to me that you were the one doing most of the work on tabs, so it is your preference.

I'm leaving this comment here because it'll bring you a laugh then.

Reply:
I added these during the dev process, which was helpful, but now we are in maintenance mode, it feels like overkill.

Reply:
lol, so you were the one that added them. I only remebered that later so came back and chuckled.
and I agree with your logic above. nice 👍